authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/pages/connectormanagement/ConnectorManagementApiService.kt

```diff
@@ -107,10 +107,13 @@ class ConnectorManagementApiService(
         deploymentEnvironmentService.assertValidEnvId(environmentId)
 
         val connectors = connectorService.getConnectorsByEnvironment(environmentId)
-        return buildConnectorOverview(connectors)
+        return buildConnectorOverview(connectors, withoutFrontendUrl = true)
     }
 
-    private fun buildConnectorOverview(connectors: List<ConnectorRecord>): ConnectorOverviewResult {
+    private fun buildConnectorOverview(
+        connectors: List<ConnectorRecord>,
+        withoutFrontendUrl: Boolean = false
```

Comment:
very nitpicky: IMO, it would read better to have the condition inverted and have a `useFrontendUrl` attribute.
Feel free to overrule me on this though.